### DISCUSSION TITLE
Calendar is an exception with 1.4.10 reflow?

### DISCUSSION THREAD

**Discussion by Ignacio-de-Loyola** [w3c/wcag#3300] [Q&A] [answered]
![image](https://github.com/w3c/wcag/assets/80868256/bdd7a50d-a581-4d82-ac0a-085d7b1f6747)
Here is example about calendar([Date Picker Dialog Example](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/)).
Horizontal scrolling is created when the browser is zoomed to 400%. 
Calendar is acceptable to provide two-dimensional scrolling? Otherwise, should the calendar be aligned vertically to eliminate horizontal scrolling?

**Comment by JAWS-test** [comment, marked as answer]
The calendar is displayed in table format. Tables are defined as an exception in 1.4.10 and are allowed to be scrollable horizontally.
